randovania/games/prime2/logic_database/Agon Wastes.txt

```diff
@@ -1988,20 +1994,37 @@ Extra - in_dark_aether: True
                   Dark World Damage ≥ 20 and Activate Safe Zone
           Standable Terrain (Intermediate) and Dark World Damage ≥ 20 and Use Screw Attack (Space Jump)
   > Event - Dark Agon Temple Key Gate
-      # https://www.youtube.com/watch?v=Tn9a6XtRnB4&list=PLPFZVy8SRe3tAkQMzVw8vWt_w_AF2_l91&index=45
+      # https://www.youtube.com/watch?v=yd91qBUxCeA
       Dark Agon Key 1 and Dark Agon Key 2 and Dark Agon Key 3 and Morph Ball Bomb and Morph Ball and Space Jump Boots and Bomb Space Jump (Expert) and Single Room Out of Bounds (Expert) and Slope Jump (Expert) and Standable Terrain (Expert) and Dark World Damage ≥ 175
+  > Amorbis Trigger
+      All of the following:
+          Knowledge (Intermediate)
+          Any of the following:
+              Space Jump Boots
+              All of the following:
+                  Morph Ball Bomb and Morph Ball
+                  Bomb Space Jump (Intermediate) or Bomb Jump (Intermediate)
+              Screw Attack at Z-Axis (Beginner) and Disabled Room Randomizer and Use Screw Attack (No Space Jump)
+              All of the following:
+                  # https://youtu.be/u9MdGBmDjNU
+                  Morph Ball Bomb and Morph Ball and After Dark Agon Energy Taken and Combat (Advanced) and Normal Damage ≥ 100 and Dark World Damage ≥ 100
+                  Bomb Jump (Beginner) or Standable Terrain (Beginner)
```

Comment:
You could probably combine this with the morph/bomb group above, but that's up to you.

Reply:
i did think of this at first, but thought it would read better if not. after doing it, it does look better grouped up.